miv/visualization/plot_burst.py

```diff
@@ -0,0 +1,46 @@
+# Bursting_Plot: Plots bursts as a bar raster with corresponding start time and duration
+##Bursting is defined as the occurence of a specified number of spikes (usually >10), with a small interspike interval (usually < 100ms)
+
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+from miv.statistics import burst
+from miv.typing import SpikestampsType
+
+
+def plot_burst(spiketrains: SpikestampsType, min_isi: float, min_len: float):
+
+    # Plots burst events across electrodes  to characterize bursting phenomenon on a singl channel
```

Comment:
Please use doctring format instead of comment

Reply:
commit 240d075f6d99006dc596e88513c270b25c544299